src/mds/MDCache.cc

```diff
@@ -5524,8 +5524,15 @@ void MDCache::export_remaining_imported_caps()
     }
   }
 
+  for (map<inodeno_t, list<MDSInternalContextBase*> >::iterator p = cap_reconnect_waiters.begin();
+       p != cap_reconnect_waiters.end();
+       ++p)
+    mds->queue_waiters(p->second);
```

Comment:
Why is this the right place to requeue any remaining cap flushes? I don't think it has much to do with exporting caps...in fact I don't think we should even have any leftover flushes, should we?